cogs5e/models/homebrew/bestiary.py

```diff
@@ -362,9 +365,20 @@ def parse_critterdb_traits(data, key):
 
         if overrides:
             for override in overrides:
-                attacks.append({'name': override.group(1) or name,
-                                'attackBonus': override.group(2) or None, 'damage': override.group(3) or None,
-                                'details': desc})
+                try:
+                    attack_yaml = yaml.safe_load(override.group(1))
+                    if not isinstance(attack_yaml, list):
+                        attack_yaml = [attack_yaml]
+                    conflicts = [a for a in attacks if a['name'].lower() in [new['name'].lower() for new in attack_yaml]]
+                    for a in conflicts:
+                        attacks.remove(a)
+                    attacks.extend(attack_yaml)
+                except Exception:
```

Comment:
`except Exception` is a very broad exception, maybe make this more specific (ex: `yaml.YAMLError`)

Reply:
Done.

I initially had it running `AttackList.from_dict(attack_yaml)` just to check early on if it was a valid attack, and stole Exception from sheetManager.py, but it seemed unnecessary...